graphql/database/experiments/features.js

```diff
@@ -308,6 +310,27 @@ const features = {
       },
     ],
   },
+  [V4_SHOW_THIRD_AD]: {
+    defaultValue: false,
+    rules: [
+      // Show on local/dev for our team only.
+      {
+        // variations: [false, true],
+        // weights: [0.1, 0.9],
+        // coverage: 1.0,
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          joined: {
+            $lt: V4_SHOW_THIRD_AD_CUTOFF_TIME,
```

Comment:
We want to include users who have joined >3 weeks ago rather than who joined after a specific datetime. This might require adding a field to the Growthbook user attributes to capture "time since joined" unless there's a MongoDB-style matcher that gets the job done here.

Reply:
Sorry, yes, your edit is what I meant to say.